NuCivic/dkan
# [CIVIC-5336] Source Harvest with machine names containing forward slash "/" will break DB queries

Issue: CIVIC-5336

## Description
Harvest Source machine names are generated from the title and used for cache directories and database table naming.
The current regex transformation is tailored toward being URL friendly which worked well except for handling the forward slash "/" character. This character is allowed and will break the database table naming schema and queries.
This PR make sure the forward slash character replacement is part of the machine name processing.

## How to reproduce

1. Create harvest source with title `Test 01/13/2017` and using any source uri.
2. Harvest the source.
3. Check the view page of any harvested dataset.
**Expected**: Dataset page is displayed.
**Actual**: Page loading fail with a WSOD.

## QA Tests
- [ ] Reproducing the issue should follow the expected behavior.
